signac/contrib/schema.py

```diff
@@ -136,6 +138,15 @@ def __str__(self):
     def __repr__(self):
         return "{}(<len={}>)".format(type(self).__name__, len(self))
 
+    def _repr_html_(self):
+        html = "{<br>"
```

Comment:
I think we should still include the type name for the project before the opening bracket.